docs/06-concepts/18-testing/04-best-practises.md

```diff
@@ -0,0 +1,120 @@
+# Best practises
+
+## Imports
+
+While it's possible to import types and test helpers from the `serverpod_test`, it's completely redundant. The generated file exports everything that is needed. Adding an additional import is just unnecessary noise and will likely also be flagged as duplicated imports by the Dart linter.
+
+### Don't
+
+```dart
+import 'serverpod_test_tools.dart';
+// Don't import `serverpod_test` directly.
+import 'package:serverpod_test/serverpod_test.dart'; ❌  
+```
+
+### Do
+
+```dart
+// Only import the generated test tools file.
+// It re-exports all helpers and types that are needed.
+import 'serverpod_test_tools.dart'; ✅ 
+```
+
+### Database clean up
```

Comment:
Suggestion: Change this to only use 2 `#`.

The right side menu on this page is a bit funky, do we really need "do" and "don't" there? maybe we could use 4 `#` to remove them? Or I think there is another way to exclude something from the menu also.

Reply:
I'll check if there's a way to hide them from the menu!